### ISSUE TITLE
999 Errors when „enable HTTPS Response“

### ISSUE THREAD

**Issue by tibor** [ProxymanApp/Proxyman#763]
Hi, I tested Proxyman the first time 2 weeks ago and everything went fine. Today I wanted to test a little bit more and I was able to get my iPhone connected to the Mac and see all requests. However, when I turned on „enable HTTPS Response“ for any domain, I would only see 999 Errors.

I deleted the certificate on both iPhone and Mac, but the problem stays the same. Do you have any idea how to fix this?

Thanks in advance!

### Proxyman version? (Ex. Proxyman 1.4.3)
2.16.1

### macOS Version? (Ex. mac 10.14)
10.14.6

### Expected behavior
Proxyman would be able to read HTTPS responses the same way as HTTP responses

**Comment by NghiaTranUIT** [comment]
Hi, can you try to reset Factory in Help -> Advanced -> Reset to see if it works?

**Comment by tibor** [comment]
Hi @NghiaTranUIT, thanks for the fast answer! Tried this but the result stays the same. I also renewed the SSL certificate on my iPhone again.

**Comment by NghiaTranUIT** [comment]
Can you try to intercept google.com from Safari on iOS to verify if you config properly

**Comment by tibor** [comment]
You mean going to google.com on my iPhone and checking what happens?

**Comment by NghiaTranUIT** [comment]
Yes. It’s. Just open Safari and try intercept google.com. If you setup correctly, you can see the response.

Here is the troubleshooting https://docs.proxyman.io/troubleshooting/get-ssl-error-from-https-request-and-response

**Comment by NghiaTranUIT** [comment]
@tibor please follow the "2.1 Verify that you're able to see other HTTPS Response without error" section on the troubleshooting page, it will guide you on how to debug what the bug is 👍

**Comment by fannar** [comment]
Having the same problem. **999 ERROR** even for https://www.google.com in mobile Safari (iOS).

**Comment by NghiaTranUIT** [comment]
Hey @fannar can you try google.com in your safari on MacBook. Does it work?

and what is your iOS version you’re using 🤔

**Comment by fannar** [comment]
Hey @NghiaTranUIT. Found out what the problem was. It was me 🤓
**General->About->Certificates Trust Settings** was **off**.
